### ISSUE TITLE
error when using srl extractor

### ISSUE THREAD

**Issue by parkcitymedia** [dsiguide/dsiguide.github.io#22]
tl;dr - the srl_dsi_extract.exe dosen't do what the guide says it should, and i'm quite stuck. (it dosen't pop out a "footer.out" file that i can edit with HxD)

when using flipnote/ugopwn to downgrade, following [this guide](https://dsiguide.me/downgrading) i'm told to download the latest version of the dsi srl extractor. after that's done, i'm told to grab the .bin file from the  `(sd card)>private>ds>title` directory and copy it into the same directory that the contents of the `srl_dsi_extract.zip` were copied into, and then drag the flipnote .bin file onto the `srl_dsi_extract.bat` and confirm something.
a console window pops up, and this is the output:
```dos
PS C:\users\username\Desktop\stuff\dsi_srl_extract> .\dsi_srl_extract.exe --basename=FLIPNOTE_STUDIO 4B475545.bin
C:\users\username\Desktop\stuff\dsi_srl_extract\dsi_srl_extract.exe for dsi by booto
decrypting tna4
tna4:
magic:    544e4134
group_id: 3031
version:  0000
mac:      78:a2:a0:b6:6d:9b
titleid:  00030004-4b475545
contents:
 tmd: 0x00000228 ebytes
 content(index:0x00, id:0x00000000): 0x002e9300 ebytes
 savedata: 0x007f0800 ebytes [0x007f0000 bytes]
decrypting footer
saving footer
checking footer signature... GOOD!
checking ap signature... GOOD!
checking tna4 sha1... GOOD!
saving tna4
decrypting banner
checking banner sha1... GOOD!
saving banner
decrypting tmd
checking tmd sha1... GOOD!
decrypting content 0
checking content sha1... GOOD!
DSiWare Title: Flipnote Studio
Nintendo
decrypting savedata
checking savedata sha1... GOOD!
unexpected bannersav elength: 0x00000000
```
i assume there might be a spelling error on that last line that has to do with something in the executeable? i don't know. 
what i receive after going through this process is a file titled `Flipnote_Studio-Nintendo.bin`, and no other file. there was no `footer.out` file in the folder.
so i got adventurous, backed up the folder, and used HxD on the `Flipnote-Studio-Nintendo.bin`, and followed the directions meant for the `footer.out` file. there was a **00000380** line on the "up and down" column, and there was a **0F** line on the "left-right" row up top. they both met at a set of characters, **00**, which when highlighted, also highlighted a period, and nothing else. this is where i've been stuck for a while. i think the issue is that the site dosen't match up with the contents of the `srl_dsi_extract.zip` file, which is causing some problems.
